src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -321,14 +291,13 @@ public ICompilationTracker FreezePartialState(CancellationToken cancellationToke
             /// returned will have a compilation that is retained so that it cannot disappear.
             /// </summary>
             private void GetPartialCompilationState(
```

Comment:
I'm guessing this can become static now? If it _can't_ that'd worry me if it means it's using state other than the one passed in.